src/SDL_log.c

```diff
@@ -39,6 +38,15 @@
 #include <android/log.h>
 #endif
 
+/* Copied from stdlib/SDL_string.c */
+#if defined(_MSC_VER) && _MSC_VER <= 1800
+/* Visual Studio 2013 tries to link with _vacopy in the C runtime. Newer versions do an inline assignment */
+#undef va_copy
+#define va_copy(dst, src)   dst = src
+#elif defined(__GNUC__) && (__GNUC__ < 3)
+#define va_copy(to, from)   __va_copy(to, from)
+#endif
```

Comment:
I'm surprised this didn't come up anywhere else in SDL. Can you put a FIXME on this to move this somewhere common (like SDL_internal.h or something)? You don't have to move it for this PR.